app-admin/etcd-wrapper/files/etcd-wrapper

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -e
+
+USAGE="
+Usage: $0 [etcd flags]
+
+Wrapper for launching etcd via rkt.
+
+The version of etcd to be used can be set through ETCD_VERSION. The version
+must be set to one of the image tags published here:
+
+  https://quay.io/repository/coreos/etcd?tab=tags
+
+Alternatively, the keywords 'alpha', 'beta', and 'stable' may be used to specify
+the version. If no version is provided, the CoreOS update group is used.
+"
+
+case ${1} in
+	-h | --help)
+		echo "${USAGE}"
+		exit 0
+		;;
+esac
+
+source /usr/share/coreos/update.conf
+source /etc/coreos/update.conf
+
+ETCD_VERSION="${ETCD_VERSION:-${GROUP:-stable}}"
+ETCD_ACI="${ETCD_ACI:-quay.io/coreos/etcd}"
```

Comment:
How are you going to publish arm64 images? ```quay.io/coreos/etcd_arm64```?  I think we need to set ETCD_ACI via a conditional on the arch.